airbyte-local-cli-nodejs/test/utils.test.ts

```diff
@@ -48,10 +48,12 @@ describe('parseConfigFile', () => {
 
 describe('checkDockerInstalled', () => {
   it('should pass if docker is installed', () => {
-    expect(checkDockerInstalled('pwd', [])).toBeUndefined();
+    (spawnSync as jest.Mock).mockReturnValue({status: 0});
```

Comment:
move non mock `node:child_process` or `node:fs` tests to `utils.it.test.ts` file